### ISSUE TITLE
Documentation Clarification of `BallotSheetId`

### ISSUE THREAD

**Issue by adghayes** [usnistgov/CastVoteRecords#37]
**Organization Name**: VotingWorks

**Organization Type**: 2, Industry

**Document (e.g., CastVoteRecords)**: CastVoteRecords

**Reference (Include section and paragraph number)**: 

**Comment (Include rationale for comment)**:

We think there may be a mistake in the documentation for a couple of the identifiers on `CVR.CVR`:

For `BallotPrePrintedId`, this description makes sense:

> A unique identifier for the ballot (or sheet of a multi-sheet ballot) that this CVR represents, used if ballots are pre-marked with unique identifiers. If provided, this number would be the same on all CVRs that represent individual sheets from the same multi-sheet ballot. This identifier is not the same as one that may be impressed on the corresponding ballot as it is scanned or otherwise associated with the corresponding ballot; see the [BallotAuditId](https://github.com/usnistgov/CastVoteRecords/blob/master/CastVoteRecords_UML_Documentation.md#_18_0_2_6340208_1469207550920_513772_4736) attribute.

But for `BallotSheetId`, the description seems incorrect: 

> A unique number for the ballot (or sheet of a multi-sheet ballot) that this CVR represents, used if ballots are pre-marked with unique numbers. If provided, this number would be the same on all CVRs that represent individual sheets from the same multi-sheet ballot. This number is not the same as one that may be impressed on the corresponding ballot as it is scanned or otherwise associated with the corresponding ballot; see the [BallotAuditId](https://github.com/usnistgov/CastVoteRecords/blob/master/CastVoteRecords_UML_Documentation.md#_18_0_2_6340208_1469207550920_513772_4736) attribute.

Since it is the `BallotSheetId`, shouldn't it be unique to each sheet? Whether it is unique within a multi-sheet ballot or universally unique is another question, but based on the name I would think that it is not the case that "this number would be the same on all CVRs that represent individual sheets from the same multi-sheet ballot." 

**Suggested Change**:

Clarify what `BallotSheetId` is meant to mean and update documentation accordingly.

---

Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by JDziurlaj** [comment]
You are correct that `BallotSheetId` corresponds roughly to the requirement identified. It’s the identifier unique to the ballot style sheet impressed on all instances. 
`BallotPrePrintedId` on the other hand is a identifier unique to a particular instance of a sheet (i.e. unique across all sheets of this ballot style).
